customization.yaml

```diff
@@ -1,24 +1,26 @@
 # you can customize with additional tag categories
 # and tags as necessary
 tag-categories:
-- name: Fybrik
-  description: Parent Category for all Fybrik labels
+- name: GenericTags
+  description: Parent Category for all generic labels
   tags:
   - name: PII
     description: Personally Identifiable Information
   - name: finance
     description: Financial Information
+- name: Purpose
+  description: The purpose for which assets may be used
+  tags:
+  - name: housing
```

Comment:
what is the housing tag ?
@simanadler probably uses it in the Kubeflow demo, but I don'y think we should add it by default.